ui/components/app/whats-new-popup/whats-new-popup.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import testData from '../../../../.storybook/test-data';
+import configureStore from '../../../store/store';
+import WhatsNewPopup from '.';
+
+export default {
+  title: 'Components/Multichain/WhatsNewPopup',
+  component: WhatsNewPopup,
+  argTypes: {
+    onClose: {
+      action: 'onClose',
+    },
+  },
+};
+
+export const DefaultStory = (args) => <WhatsNewPopup {...args} />;
+DefaultStory.decorators = [
+  (Story) => (
+    <Provider store={configureStore(testData)}>
+      <Story />
+    </Provider>
+  ),
+];
```

Comment:
All of the stories are already wrapped with a redux provider and fed the test-data state so it should still work without this. You could just pass the state that you are adding to the test data like we do here https://github.com/MetaMask/metamask-extension/blob/9a939f69637473d45e4391e47b080772d5432fcc/ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.stories.js#L11-L24

Reply:
Oh wow, I didn't know that!  Thank you!